api/src/services/script-runner.service.ts

```diff
@@ -605,6 +616,153 @@ export class ScriptRunnerService {
+      );
+    }
+
+    const take = calculateTake(40);
+    const skip = calculateSkip(take, dataTransferDTO.page || 1);
+    const listingTransferMap = await this.prisma.listingTransferMap.findMany({
+      take,
+      skip,
+    });
+    console.log(
+      `Found ${listingTransferMap.length} listings on page ${
+        dataTransferDTO.page || 1
+      } out of a possible 40 for this page`,
+    );
+    // loop over each new listing id <-> old listing id relation
+    for (let i = 0; i < listingTransferMap.length; i++) {
+      const oldAssetInfo: {
+        ordinal: number;
+        created_at: Date;
+        updated_at: Date;
+        file_id: string;
+        label: string;
+      }[] = await client.$queryRaw`SELECT
+            li.ordinal,
+            a.created_at,
+            a.updated_at,
+            a.file_id,
+            label
+        FROM listing_images li
+            JOIN assets a ON a.id = li.image_id
+        WHERE li.listing_id = ${listingTransferMap[i].oldId} :: UUID
+          AND a.file_id IS NOT NULL AND a.file_id != ''`;
+      console.log(
+        `moving ${oldAssetInfo.length || 0} assets for listing: ${
+          listingTransferMap[i].oldId
+        }:`,
+      );
+      // loop over each listing image on the old listing
+      for (let j = 0; j < oldAssetInfo.length; j++) {
+        // pull down image from cloudinary
+        const image = await axios.get(
+          `https://res.cloudinary.com/${dataTransferDTO.cloudinaryName}/image/upload/${oldAssetInfo[j].file_id}.jpg`,
+          {
+            responseType: 'arraybuffer',
+          },
+        );
+        const newFileId = (oldAssetInfo[j].file_id as string)
+          .replace('housingbayarea/', '')
+          .replace('dev/', '');
+
+        // upload image to s3
+        const res = await this.assetService.upload(newFileId, {
+          filename: null,
+          buffer: image.data,
+          fieldname: null,
+          originalname: `${newFileId}.jpg`,
+          encoding: null,
+          mimetype: 'image/jpeg',
+          size: image.data.length,
+          destination: null,
+          path: null,
+          stream: null,
+        });
+
+        // update new listing with these assets
+        await this.prisma.listings.update({
+          where: {
+            id: listingTransferMap[i].listingId,
+          },
+          data: {
+            listingImages: {
+              create: {
+                assets: {
+                  create: {
+                    fileId: res.url,
+                    label: 'cloudinaryBuilding',
+                  },
+                },
+                ordinal: oldAssetInfo[j].ordinal,
+              },
+            },
+          },
+        });
+      }
+    }
+
+    // disconnect from foreign db
+    await client.$disconnect();
+
+    // script runner standard spin down
+    await this.markScriptAsComplete(
+      `data transfer assets ${dataTransferDTO.jurisdiction}`,
+      requestingUser,
+    );
```

Comment:
issue: This needs to be updated to be the same as the `markScriptAsRunStart` otherwise a 500 error throws